empire.go

```diff
@@ -141,6 +145,10 @@ func (opts CreateOpts) Event() CreateEvent {
 
 // Create creates a new app.
 func (e *Empire) Create(ctx context.Context, opts CreateOpts) (*App, error) {
+	if err := e.MaintenanceMode; err != nil {
```

Comment:
What about a HttpHandler func that we can wrap 'write' methods with?  Less boilerplate, and it's explicit at the routing layer what is possible for blocking with maintenance mode.  Not sure that works well with a generic ACL system though.